.evergreen/release-build-papertrail-vars.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -o errexit
+set +x
+
+. ${DRIVERS_TOOLS}/.evergreen/secrets_handling/setup-secrets.sh drivers/rust
```

Comment:
This makes the AWS secrets manager the source of these values rather than being hand-configured Evergreen variables, which should be easier to maintain.